vision/standard_object/odlc_text_detection.py

```diff
@@ -0,0 +1,132 @@
+"""
+Method for reading the text on the odlc standard objects.
+"""
+
+from typing import Any, Dict, List
+
+import sys
+import cv2
+import numpy as np
+import numpy.typing as npt
+import pytesseract
+
+from vision.common.bounding_box import ObjectType, BoundingBox
+
+# from bounding_box import ObjectType, BoundingBox  # A local file is required when testing
+
+
+def text_detection_pre_processing(unprocessed_img: npt.NDArray[np.uint8]) -> npt.NDArray[np.uint8]:
+    """
+    This does the preprocessing effects on the passed image needed to make text detection easier.
+
+    Parameters
+    ----------
+    img: npt.NDArray[np.uint8]
+        The image that you want to do the preprocessing on.
+
+    Returns
+    ----------
```

Comment:
Make sure the number of dashes equals the number of letters in the header
```suggestion
    Returns
    -------
```